src/api/cf.nim

```diff
@@ -26,12 +27,20 @@ const
   ## base url of the forgesvc endpoint
   modsBaseUrl = "https://addons-ecs.forgesvc.net/api/v2"
 
+
+converter getDependencies(json: JsonNode): int =
+  var id: int = json{"id"}.getInt()
```

Comment:
You don't need to create an `id` variable, you can assign to `result` directly.
https://nim-by-example.github.io/variables/result/